src/python/grpcio/grpc/_common.py

```diff
@@ -15,6 +15,7 @@
 
 import logging
 
+import time
```

Comment:
Should we sort the imports for the files we touched?

Reply:
This is the order prescribed by pylint:
```
src/python/grpcio/grpc/_common.py:19:0: C0411: standard import "import time" should be placed before "import six" (wrong-import-order)
```